devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy

```diff
@@ -355,7 +361,7 @@ def zwaveEvent(physicalgraph.zwave.commands.meterv3.MeterReport cmd, ep = null)
 			eventMap.value = Math.round(cmd.scaledMeterValue)
 			eventMap.unit = "W"
 			events += createEvent(eventMap)
-			if (cmd.scaledMeterValue) {
+			if (Math.round(cmd.scaledMeterValue)) {
```

Comment:
is this because you were getting like .01W reports when the device was actually stopped?

Reply:
exactly